google/cloud/bigquery/v2/minimal/internal/job_logging.cc

```diff
@@ -65,17 +73,21 @@ StatusOr<GetJobResponse> BigQueryJobLogging::GetJob(
              GetJobRequest const& request) {
         return child_->GetJob(rest_context, request);
       },
-      rest_context, request, __func__, tracing_options_);
+      rest_context, request, __func__, "GetJobRequest", "GetJobResponse",
```

Comment:
The protobuf formatter uses `m.GetTypeName()`, which renders the fully-qualified type name.  This should follow the same pattern.